complaints/ccdb/verify_s3.py

```diff
@@ -0,0 +1,168 @@
+from __future__ import print_function
+
+import os
+import sys
+from zipfile import ZipFile
+
+import boto3
+import configargparse
+import requests
+
+# -----------------------------------------------------------------------------
+# Process
+# -----------------------------------------------------------------------------
+
+
+class ProgressPercentage(object):
+    def __init__(self, options):
+        self.options = options
+        self.seen_so_far = 0
+
+    def __call__(self, bytes_amount):
+        self.seen_so_far += bytes_amount
+        sys.stdout.write(
+            "\r{}  {:,d} bytes".format(self.options.infile, self.seen_so_far)
+        )
+        sys.stdout.flush()
+
+
+def verify_download(options):
+    # Setup
+    temp_json_file_name = "todays_json_export.json"
+    size_file_path = "complaints/ccdb/json_prev_size.txt"
```

Comment:
Hardcoded strings should be part of configuration, even if they are set as default values

You could also derive these from `options.json_file`

Reply:
The `verify_download` function was from my first iteration before the scarecrow discussion. I only left it in there in case we ended up needing to download the files for comparison. I'll add some comments explaining that